### PR TITLE
fixes(#125)

### PR DESCRIPTION
**Pull request by MidSpike**

### CODE REVIEW

**Review by TotoTheEgg — APPROVED** [review]

**Review by WillFJS — APPROVED** [review]

**Comment by WillFJS** [comment]
This is for totos project correct?

**Comment by TotoTheEgg** [comment]
@WillFJS yes it is for my project

**Review by dr4wn — APPROVED** [review]
looks good

**Review by TotoTheEgg — APPROVED** [review]

**Review by TotoTheEgg — APPROVED** [review]

**Review by rossloika — APPROVED** [review]

File: src/server/v1/auth/tokens/test.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        if (!game_owner_id || typeof game_owner_id !== 'string') {
+            return res.status(400).send(JSON.stringify({
+                'message': 'missing (string) \`game_owner_id\` in request body',
+            }, null, 2));
+        }
+
+        /* fetch user-blacklist-info */
+        const [ db_blacklisted_user_data ] = await go_mongo_db.find(process.env.MONGO_DATABASE_NAME, process.env.MONGO_BLACKLISTED_USERS_COLLECTION_NAME, {
+            'identity.roblox_user_id': game_owner_id,
+        });
```
Comment (MidSpike) [inline src/server/v1/auth/tokens/test.js:49]:
```suggestion
        }).catch(() => []);
```

File: src/server/v1/auth/tokens/test.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        if (db_blacklisted_user_data) {
+            return res.status(403).send(JSON.stringify({
+                'message': '\`game_owner_id\` is blacklisted!',
+            }, null, 2));
+        }
+
+        /* fetch user-auth-info */
+        const [ db_user_auth_data ] = await go_mongo_db.find(process.env.MONGO_DATABASE_NAME, process.env.MONGO_API_AUTH_USERS_COLLECTION_NAME, {
+            'identity.roblox_user_id': game_owner_id,
+        });
```
Comment (MidSpike) [inline src/server/v1/auth/tokens/test.js:61]:
```suggestion
        }).catch(() => []);
```

**Pull request merged**